english/staypuft/place-hat/place-hat.md

```diff
@@ -1 +1,7 @@
-The marshmallow man springs to life and exclaims "Happy Birthday!"
\ No newline at end of file
+The marshmallow man springs to life and exclaims "Happy Birthday!"
+
+"How did you know", you reply.
+
+["I am your father's brother's nephew's cousin's former roommate." replied the snowman.](how-did-you-know-roommate/roommate.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.